neuron.cpp

```diff
@@ -321,6 +321,50 @@ bool Neuron::loadModbusMap()
     return true;
 }
 
+bool Neuron::modbusWriteRequest(QUuid requestId, QModbusDataUnit request)
+{
+    if (!m_modbusInterface)
+        return false;
+
+    if (QModbusReply *reply = m_modbusInterface->sendWriteRequest(request, m_slaveAddress)) {
+        if (!reply->isFinished()) {
+            connect(reply, &QModbusReply::finished, this, [reply, requestId, this] {
+                reply->deleteLater();
+
+                if (!m_writeRequestQueue.isEmpty()) {
+                    QPair<QUuid, QModbusDataUnit> request = m_writeRequestQueue.first();
+                    modbusWriteRequest(request.first, request.second);
```

Comment:
hmm... I fail to see when you remove a processed request from the queue... what am I missing?